docker-entrypoint.sh

```diff
@@ -12,10 +12,12 @@ case "$1" in
 
         echo "Starting gunicorn with config: $_SETTINGS_FILE"
 
+        # the 'gevent' worker class causes gunicorn to fail to load
+        # under Docker. Switching to 'sync' solves this.
         exec gunicorn \
             --paste "$_SETTINGS_FILE" \
             --bind ${HOST-127.0.0.1}:${PORT-8000}\
-            --worker-class gevent \
+            --worker-class sync \
```

Comment:
This is a significant change, does it feel like something we might be able to fix with some investigating, or is it a fundamental limit of switching to pypy?

Reply:
There appear to be several incompatibilities with this combination of gevent under pypy. However, `sync` under pypy is as performant as gevent. 

I'll dig into this a bit more to see if there's a good option.

Reply:
I have no objection to switching to the `sync` backend, we'll just need to have a good loadtesting/monitoring strategy for if it behaves differently in the wild.

Reply:
understood. This is also partly to resolve issues with CI. It looks like [gevent 1.5 may have a fix](https://github.com/gevent/gevent/commit/d0e0465891e30abb3e64c25324268251ad5c994c) for this, when it's available.